targets/TARGET_NORDIC/README.md

```diff
@@ -0,0 +1,55 @@
+# Nordic platform's readme
+
+This readme file contains several hints useful for any Nordic's user.
+
+## 1. Nordic�s development kit targets
+
+Nordic provides several development kit targets, called *[base target]*. For example:
+- NRF51_DK
+- NRF51_DK_LEGACY
+- NRF52_DK
+- NRF51_DONGLE
+- others are incoming as mbed-os is under continuous construction!
```

Comment:
Nordic continuously supports mbed and other targets will become available with releases of new Nordic development kits.
?